src/utils/wallet.js

```diff
@@ -168,7 +163,7 @@ class Wallet {
         return ACCOUNT_ID_REGEX.test(accountId);
     }
 
-    async sendMoney(receiverId, amount) {
+    async sendMoney({ account, receiverId, amount }) {
```

Comment:
Not a blocked. You added `account` prop but it's not used. But I guess it can be removed since you are using, `account.sendMoney` in `FungibleTokens` class. We can also remove the action from `actions/account.js`

Reply:
Nice catch, I didn't mean to change this function signature at all, and this change probably broke the existing `send` container flow by changing it's signature which isn't intended yet (just in case we need to toggle back to the original send container without reverting all of this code). Fixed!